extensions/scarb-doc/src/lib.rs

```diff
@@ -7,16 +7,18 @@ use types::Crate;
 
 pub mod compilation;
 pub mod db;
+pub mod docs_generation;
 pub mod types;
 pub mod versioned_json_output;
 
 pub fn generate_language_elements_tree_for_package(
     package_name: String,
     project_config: ProjectConfig,
 ) -> Crate {
-    let db = ScarbDocDatabase::new(Some(project_config));
+    let crate_name = package_name;
```

Comment:
unnecessary variable?

Reply:
Yes and no - the semantic changes here, it starts being crate name instead of package name, even though they are equal

Reply:
After code changes it indeed makes no sense